src/Translation/TranslationDelegate.php

```diff
@@ -15,6 +16,35 @@ public function __construct(TranslationApiInterface ...$apis)
     $this->apis = $apis;
   }
 
+  /**
+   * @throws InvalidArgumentException
+   */
+  public function translateProject(Program $project, ?string $source_language, string $target_language): ?array
```

Comment:
It's refactored this way so that TranslationDelegate can log project id in the future